### PR TITLE
ADD sale_layout_multi_company

### PR DESCRIPTION
**Pull request by eLBati**
This module extends the functionality of sale_layout module adding support for multi company

### CODE REVIEW

File: sale_layout_multi_company/README.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+This module extends the functionality of sale_layout module adding support for multi company
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/167/8.0
+
+Known issues / Roadmap
```
Comment (pedrobaeza) [inline sale_layout_multi_company/README.rst]: Remove this empty section

File: sale_layout_multi_company/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Lorenzo Battistini - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Layout - Multi company",
+    "summary": "Multi company features for sale_layout",
+    "version": "8.0.1.0.0",
+    "category": "Sale",
```
Comment (pedrobaeza) [inline sale_layout_multi_company/__openerp__.py]: Sales Management

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: sale_layout_multi_company/README.rst (hunk 1 above)
Comment (eLBati) [reply sale_layout_multi_company/README.rst]: done

File: sale_layout_multi_company/__openerp__.py (hunk 2 above)
Comment (eLBati) [reply sale_layout_multi_company/__openerp__.py]: done

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
When CIs goes green :+1:

**Comment by archetipo** [comment]
:+1:

**Comment by eLBati** [comment]
merge?

**Pull request merged**